### DISCUSSION TITLE
Node

### DISCUSSION THREAD

**Discussion by BullForAReason** [availproject/avail#352] [General]
How to run node properly ?

**Comment by QEDK** [comment]
Have you checked out https://docs.availproject.org/category/run-a-full-node?

**Reply by BullForAReason** [reply]
Heyy Thanks for the reply I have successful Done with the node..
But i have one thing in my mind is that i'm running my node on terminus What if i shut down my pc should my node keep going ?

**Reply by QEDK** [reply]
If your node is running on the PC, yes it will shut down along with the machine, if you want to keep your node synced at all times, you should consider a cloud instance provider like AWS or VPS providers like OVH.

**Reply by BullForAReason** [reply]
Can we run on mobile via Terminus mobile app ?

**Reply by BullForAReason** [reply]
I bought an cloud server for 1 year

**Reply by QEDK** [reply]
You can just run it on the cloud server, and you can SSH from your mobile if needed.

**Reply by BullForAReason** [reply]
Bro can u guide me a bit via telegram
if possible

**Reply by BullForAReason** [reply]
I bought server from Contabo

**Reply by QEDK** [reply]
Unfortunately due to the number of help requests we receive from various channels on a daily basis, we are unable to offer dedicated support for each person, if you can expand further here, me or someone from my team will be happy to help you! Hope that makes sense 😄

**Reply by BullForAReason** [reply]
Can u guide me proper way to run an node so it won't get stopped I want to contribute to this ecosystem

**Reply by BullForAReason** [reply]
bro u their ?

**Reply by BullForAReason** [reply]
can u guide me how to run node properly from long run ? without interupting it

**Comment by markopoloparadox** [comment]
At the moment, we do not provide any assistance for setting up the infrastructure for running the node. This is something that you as a node operator are supposed to already know. 

As per our guide, you need to have some basic system administration knowledge to properly set up everything otherwise you will face a lot of difficult challenges.

At this point, I would suggest you to get informed on how the systemd service works as it can help you to achieve what you want.

**Comment by BullForAReason** [comment]
please share avail discord link I'm unable to join
